spring-boot-modules/spring-boot-groovy/pom.xml

```diff
@@ -39,6 +40,12 @@
             <artifactId>h2</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
```

Comment:
Try removing explicit rest-assured version - it already comes transitively from spring-boot-dependencies of parent-boot-3